src/modules/Extension/html_admin/mod_extension_languages.html.twig

```diff
@@ -1,73 +1,158 @@
+                                <th>{{ 'Language'|trans }}</th>
+                                <th>{{ 'Completion'|trans}}</th>
+                                <th>{{ 'Code'|trans }}</th>
+                                <th>{{ 'Enable'|trans }}</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            {% for lang in admin.extension_languages({'disabled': true}) %}
+                                <tr>
+                                    <td class="w-2">
+                                        <span class="flag flag-country-{{ lang.locale|split('_')[1]|lower }}"></span>
+                                    </td>
+                                    <td class="w-6">{{ lang.name }}</td>
+                                    <td>
+                                        {% set progress = admin.extension_locale_completion({'locale_id': lang.locale})%}
+                                        {% if progress > 75 %}
+                                            {% set background = 'success' %}
+                                        {% elseif progress > 50 %}
+                                            {% set background = 'warning' %}
+                                        {% else %}
+                                            {% set background = 'danger' %}
+                                        {% endif %}
+                                        <div class="progress" style="height: 15px;">
+                                            <div class="progress" style="height: 15px;">
+                                                <div class="progress-bar bg-{{ background }}" role="progressbar" style="width: {{ progress }}%" aria-valuenow="{{ progress }}" aria-valuemin="0" aria-valuemax="100">{{ progress }}%</div>
+                                            </div>
+                                        </div>
+                                    </td>
+                                    <td class="w-1">{{ lang.locale }}</td>
+                                    <td class="w-1">
+                                        <a class="btn btn-icon api-link"
+                                            href="{{ 'api/admin/extension/toggle_language'|link({ 'locale_id': lang.locale }) }}"
+                                            data-api-reload="1">
+                                            <svg class="icon">
+                                                <use xlink:href="#check" />
+                                            </svg>
+                                        </a>
+                                    </td>
+                                </tr>
+                            </tbody>
+                            {% else %}
+                                <tbody>
+                                <tr>
+                                    <td class="text-muted" colspan="2">{{ 'The list is empty'|trans }}
```

Comment:
Same here e.g. "There are currently no languages that are disabled".( or "can be enabled.")